packages/build-runtime.js

```diff
@@ -70,8 +70,8 @@ each(File.helpers, function (helperName) {
   writeFile("helpers/" + helperName + ".js", buildHelper(helperName));
 });
 
-writeFile("regenerator/index.js", readFile("regenerator-babel/runtime-module", true));
-writeFile("regenerator/runtime.js", selfContainify(readFile("regenerator-babel/runtime")));
+writeFile("regenerator/index.js", readFile("../node_modules/regenerator/runtime-module", true));
```

Comment:
`../node_modules` isn't necessary and it'll break if it's installed elsewhere. The paths are ran through `require.resolve`.